src/Infrastructure/Scout/ScoutSearchCommandBuilder.php

```diff
@@ -2,18 +2,21 @@
 
 declare(strict_types=1);
 
-namespace JeroenG\Explorer\Application;
-
-use JeroenG\Explorer\Domain\Compound\BoolQuery;
-use JeroenG\Explorer\Domain\Compound\CompoundSyntaxInterface;
+namespace JeroenG\Explorer\Infrastructure\Scout;
```

Comment:
Why should this be in infrastructure?

Reply:
Because it builds a command specifically for scout